### DISCUSSION TITLE
Is there a way to get instance segmentation for more than 256 objects in the environment?

### DISCUSSION THREAD

**Discussion by meidachen** [microsoft/AirSim#3795] [Support Q&A] [answered]

**Comment by meidachen** [comment]
Is there a way to get instance segmentation for more than 256 objects in the environment?

**Comment by rajat2004** [comment, marked as answer]
This is a UE4 limitation currently ([Docs](https://docs.unrealengine.com/4.26/en-US/BlueprintAPI/Rendering/SetCustomDepthStencilValue/)), since segmentation uses custom stencil buffer with 1 byte per pixel. See https://github.com/microsoft/AirSim/issues/2252 as well
